src/Windowing/Silk.NET.GLFW/GLFW.cs

```diff
@@ -0,0 +1,421 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+using AdvancedDLSupport;
+using Silk.NET.Core.Loader;
+using Silk.NET.Core.Native;
+
+namespace Silk.NET.GLFW
+{
+    /// <summary>
+    /// Provides access to the GLFW API.
+    /// </summary>
+    public abstract class GLFW : NativeAPI, IGLFW
```

Comment:
Can we rename this to `Glfw` so that the class name doesn't conflict with the namespace name?

Reply:
Sure. Should IGLFW become IGlfw too?

Reply:
Yes please!

Reply:
The other types that have `GLFW` in the name too? Callbacks, Exception, LibraryNameContainer, and Provider

Reply:
Errrr yeah rename them all.